lib/generators/react_on_rails/templates/client/server.js

```diff
@@ -0,0 +1,42 @@
+/* eslint-disable no-console, func-names, no-var */
+var bodyParser = require('body-parser');
+var webpack = require('webpack');
+var WebpackDevServer = require('webpack-dev-server');
+var jade = require('jade');
+var sleep = require('sleep');
+var config = require('./webpack.client.hot.config');
+
+var comments = [
+  { author: 'Pete Hunt', text: 'Hey there!' },
+  { author: 'Justin Gordon', text: 'Aloha from @railsonmaui' },
+];
+
+var server = new WebpackDevServer(webpack(config), {
+  publicPath: config.output.publicPath,
+  hot: true,
+  historyApiFallback: true,
+  stats: {
+    colors: true,
+    hash: false,
+    version: false,
+    chunks: false,
+    children: false,
+  },
+});
+
+server.app.use(bodyParser.json(null));
+server.app.use(bodyParser.urlencoded({extended: true}));
+
+server.app.use('/', function(req, res) {
+  var locals = {
+    props: JSON.stringify(comments),
+  };
+  var layout = process.cwd() + '/index.jade';
+  var html = jade.compileFile(layout, { pretty: true })(locals);
+  res.send(html);
+});
+
+server.listen(3000, 'localhost', function(err) {
```

Comment:
FYI, we might change this default: https://github.com/shakacode/react-webpack-rails-tutorial/issues/116